repository/s.json

```diff
@@ -3933,6 +3933,25 @@
 			]
 		},
 		{
+			"name": "SublimeCodeIntel",
+			"author": "Kronuz",
+			"description": "Full-featured code intelligence and smart autocomplete engine",
+			"details": "https://github.com/SublimeCodeIntel/SublimeCodeIntel",
+			"homepage": "http://sublimecodeintel.github.io/SublimeCodeIntel/",
```

Comment:
It redirects to https anyway, but there's no reason not to specify that directly.